tensorflow/java/src/test/java/org/tensorflow/EagerOperationBuilderTest.java

```diff
@@ -48,5 +48,92 @@ public void failToBuildOpIfSessionIsClosed() {
     }
   }
 
-  // TODO (karllessard) add more tests when EagerOperation is implemented as well
+  @Test
+  public void addInputs() {
+    try (EagerSession session = EagerSession.create()) {
+      Operation asrt = opBuilder(session, "Assert", "assert")
+          .addInput(TestUtil.constant(session, "Cond", true))
+          .addInputList(new Output<?>[] {TestUtil.constant(session, "Error", -1)})
+          .build();
+      try {
+        opBuilder(session, "Const", "var").addControlInput(asrt);
+        fail();
+      } catch (UnsupportedOperationException e) {}
```

Comment:
failing internal lint test , here is the error: Empty catch block. [EmptyCatchBlock]